### PR TITLE
add: rngd to cse install for ubuntu

### PR DESCRIPTION
**Pull request by alexeldeib**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ installDeps() {
       done
     fi
 
-    for apt_package in apache2-utils apt-transport-https ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers iotop iproute2 ipset iptables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysfsutils sysstat traceroute util-linux xz-utils netcat dnsutils zip; do
+    for apt_package in apache2-utils apt-transport-https blobfuse=${BLOBFUSE_VERSION} ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers iotop iproute2 ipset iptables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysfsutils sysstat traceroute util-linux xz-utils netcat dnsutils zip rng-tools; do
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:44]: blobfuse=${BLOBFUSE_VERSION} ? maybe stray diff from rebase

Comment (UtheMan) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh]: fixed

**Review by alexeldeib — COMMENTED** [review]
thanks @UtheMan!

lgtm but can't approve since I originally opened the PR 😄

**Review by UtheMan — APPROVED** [review]
LGTM

**Pull request merged**